README.md

```diff
@@ -92,8 +92,7 @@ Depending on your Home Assistant's configuration, your weather provider might de
   date_pattern: P
   hide_today_section: false
   hide_forecast_section: false
-  hide_clock: false
```

Comment:
was this deletion intentional?

Reply:
No, I have overlooked it while merging. Have added it again.